python-frontend/src/main/java/org/sonar/python/semantic/v2/SymbolsModuleTypeProvider.java

```diff
@@ -47,9 +49,8 @@ public SymbolsModuleTypeProvider(ProjectLevelSymbolTable projectLevelSymbolTable
     this.projectLevelSymbolTable = projectLevelSymbolTable;
   }
 
-  public void createBuiltinModule(ModuleType parent) {
-    var name = "builtins";
-    createModuleFromSymbols(name, parent, TypeShed.builtinSymbols().values());
+  public ModuleType createBuiltinModule() {
```

Comment:
I'm not sure yet that I see the value of removing the `builtins` prefix.

I believe we could still have a `builtins` dedicated module with a helper method that would help retrieve it so that we don't have to duplicate the `"builtins"` literal everywhere?

Reply:
it is mainly needed to convert symbols (old ones) to types, the builtin types don't have `builtins` in their FQN

Reply:
I see! I think it was a mistake on our end in the original implementation, though. Since other tools (including the Python interpreter itself) will map builtin types to the `builtins` module.

Ideally, I think I'd like to follow that convention as well, however I think for now the least disruptive approach is okay (i.e, if this PR still achieves its goals without removing `builtins`, I'd keep the prefix - and if it breaks a lot of other stuff, we can consider adding it back later instead).